engine/src/main/java/org/terasology/rendering/cameras/Camera.java

```diff
@@ -203,19 +211,18 @@ public Vector3f getViewingDirection() {
 
     /**
      * Get the orientation of the camera.
-     * @return the orientation direction, a quaternion.
+     * @return the orientation
      */
     public Quat4f getOrientation() {
-        return new Quat4f(JomlUtil.from(viewingDirection), viewingAngle);
+        return new Quat4f(from(viewingAxis), viewingAngle);
```

Comment:
would it be more clear to use JomlUtil.from instead of importing the static method.

Reply:
alright